src/async-components/views/dialogs/security/CreateSecretStorageDialog.js

```diff
@@ -22,7 +22,7 @@ import {MatrixClientPeg} from '../../../../MatrixClientPeg';
 import FileSaver from 'file-saver';
 import {_t, _td} from '../../../../languageHandler';
 import Modal from '../../../../Modal';
-import { promptForBackupPassphrase } from '../../../../SecurityManager';
+import { getSecretStorageKeyFromAppStorage, promptForBackupPassphrase } from '../../../../SecurityManager';
```

Comment:
linter is complaining about this

Reply:
Ah thanks, left overs from a previous attempt, should be fine after removing.